coralproject/talk
# Configure Styles and UX Polish

## What does this PR do?
- We did a complete style and UX polish to the Configure Tab

## How do I test this PR?

- Go to admin
- Go to the configure tab
- Make sure that everything works as expected <3
